File: src/main/java/com/example/demo/lib/Huga.java
package com.example.demo.lib;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.PropertyNamingStrategy;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.AllArgsConstructor;
import lombok.Data;

@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
@Data
@AllArgsConstructor(staticName = "of")
public class Huga {
    private int id;

//    @JsonProperty("c_value")
    private float cValue;

    private String beType;
}

File: src/main/java/com/example/demo/lib/Huga.kt
package com.example.demo.lib

import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.databind.PropertyNamingStrategies
import com.fasterxml.jackson.databind.annotation.JsonNaming

@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy::class)
data class Huga(
    var id: Int,
    @JsonProperty("c_value") var cvalue: Float,
    var beType: String) {
    companion object {
        fun dead(): Huga = Huga(100, 1.23864F, "俺は死んじまった打")
    }
}
